__tests__/parse-templates.test.ts

```diff
@@ -62,37 +62,14 @@ describe('parseTemplates', function () {
     `);
   });
 
-  it('hbs`Hello!`', function () {
+  it('hbs`Hello!` when only matching <template>', function () {
     const input = 'hbs`Hello!`';
 
     const templates = parseTemplates(input, 'foo.js', {
       templateTag: 'template',
     });
 
-    expect(templates).toMatchInlineSnapshot(`
-      Array [
-        Object {
-          "end": Object {
-            "0": "\`",
-            "1": undefined,
-            "groups": undefined,
-            "index": 10,
-            "input": "hbs\`Hello!\`",
-          },
-          "importIdentifier": "hbs",
-          "importPath": undefined,
-          "start": Object {
-            "0": "hbs\`",
-            "1": "hbs",
-            "groups": undefined,
-            "index": 0,
-            "input": "hbs\`Hello!\`",
-          },
-          "tagName": "hbs",
-          "type": "template-literal",
-        },
-      ]
-    `);
```

Comment:
This assertion was from an older version of the implementation (back when we just parsed all template literals regardless of import, and we have been preserving the tests along and along), but it is fundamentally invalid. If you do not pass `templateLiteral` options to `parseTemplates` you should **absolutely not** get any results back for template literals.